src/doc/en/developer/doctesting.rst

```diff
@@ -1350,6 +1353,22 @@ utilized most efficiently)::
     Doctesting 2067 files using 2 threads.
     ...
 
+At the end of the doctest run, Sage updates the json file if it exists or creates
+a new one.
+
+The recorded pass/fail status of the files can be used for running only those files
+that failed their most recent test by using the ``--failed`` flag (``-f`` for short).
+
+Using the option ``--baseline-stats-path known-test-failures.json``,
+it is possible to distinguish files with known doctest failures
+from new failures. The file ``known-test-failures.json`` should be
+prepared in the same format as ``timings2.json``.
+
+Source files that marked as failed there will be marked as "[failed in baseline]"
+failures in the doctest report; and if there are only baseline failures, no
```

Comment:
`Source files marked as failed there...` or `Source files that are marked as failed there`?

Reply:
Fixed in 970f1df593e